src/XeroPHP/Remote/Query.php

```diff
@@ -41,22 +55,64 @@ public function from($class) {
     }
 
     /**
+     * @param string $where
+     * @param mixed|null $value
      * @return $this
      */
-    public function where() {
-        $args = func_get_args();
-
-        if(func_num_args() === 2) {
-            $this->where[] = sprintf('%s=="%s"', $args[0], $args[1]);
+    public function where($where, $value = null) {
+        if ($where && $value) {
```

Comment:
What is this actually testing with ```$where``` and ```$value``` and what if you want to pass ```0``` as a condition?  Please use strict type comparisons everywhere for clarity.  (I don't even think you need to be testing $where here as it's a mandatory parameter).  A better test would be ```if($value !=== null)```

Also, it seems a little risky sha1'ing the value - what happens if two clauses have the same value?